### ISSUE TITLE
how to custom learning rate decay policy ?

### ISSUE THREAD

**Issue by silverlining21** [tensorflow/tensorflow#6471]
as it shown in the learning rate deay **[API doc](https://www.tensorflow.org/api_docs/python/train/decaying_the_learning_rate)** , there are only 4 kinds of policies. but in caffe  policies are as follows:  
 
1.   //    - fixed: always return base_lr.  
2.   //    - step: return base_lr * gamma ^ (floor(iter / step))  
3.   //    - exp: return base_lr * gamma ^ iter  
4.   //    - inv: return base_lr * (1 + gamma * iter) ^ (- power)  
5.   //    - multistep: similar to step but it allows non uniform steps defined by  
6.   //      stepvalue  
7.   //    - poly: the effective learning rate follows a polynomial decay, to be  
8.   //      zero by the max_iter. return base_lr (1 - iter/max_iter) ^ (power)  
9.   //    - sigmoid: the effective learning rate follows a sigmod decay  

in issue #2922 they mentioNed on write a warpper for the existing method, but they did not implemented anything new ?  for example the **inv** `[base_lr * (1 + gamma * iter) ^ (- power)]` policy in caffe.

so I wonder if there anyone who can help me out how to custom learning rate decay policy by myself, any suggestions will be sincerely gratitude.

**Comment by aselle** [comment]
This question is best asked on Stack Overflow which is geared toward helping users of TensorFlow with usage questions. GitHub is concerned mainly with bugs and feature requests. Thanks!
